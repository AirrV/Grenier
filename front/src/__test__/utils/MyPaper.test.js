import React from 'react';
import renderer from "react-test-renderer";
import MyPaper from "../../components/utils/MyPaper.js";

it("Renders the welcome page", () => {
  const tree = renderer
    .create(<MyPaper />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})