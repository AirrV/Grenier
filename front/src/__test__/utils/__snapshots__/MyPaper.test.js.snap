// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the welcome page 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation1-5 MyPaper-paper-1"
/>
`;
